src/components/dataset/entities.vue

```diff
@@ -13,23 +13,27 @@ except according to the terms contained in the LICENSE file.
   <div id="dataset-entities">
     <page-section>
       <template #heading>
-        <span>{{ $t('resource.entities') }}</span>
-        <button v-if="project.dataExists && project.permits('entity.create')"
-          id="dataset-entities-upload-button" type="button"
-          class="btn btn-primary" @click="upload.show()">
-          <span class="icon-upload"></span>{{ $t('action.upload') }}
-        </button>
-        <template v-if="deletedEntityCount.dataExists">
-          <button v-if="canDelete && (deletedEntityCount.value > 0 || deleted)" type="button"
-            class="btn toggle-deleted-entities" :class="{ 'btn-danger': deleted, 'btn-link': !deleted }"
-            @click="toggleDeleted">
-            <span class="icon-trash"></span>{{ $tcn('action.toggleDeletedEntities', deletedEntityCount.value) }}
-            <span v-show="deleted" class="icon-close"></span>
+        <div class="dataset-entities-heading-row">
+          <button v-if="project.dataExists && project.permits('entity.create')"
+            id="dataset-entities-upload-button" type="button"
+            class="btn btn-primary" @click="upload.show()">
+            <span class="icon-upload"></span>{{ $t('upload') }}
           </button>
-        </template>
-        <odata-data-access :analyze-disabled="deleted"
-          :analyze-disabled-message="$t('analyzeDisabledDeletedData')"
-          @analyze="analyze.show()"/>
+          <template v-if="deletedEntityCount.dataExists">
+            <button v-if="canDelete && (deletedEntityCount.value > 0 || deleted)" type="button"
+              class="btn toggle-deleted-entities" :class="{ 'btn-danger': deleted, 'btn-link': !deleted }"
+              @click="toggleDeleted">
+              <span class="icon-trash"></span>{{ $tcn('action.toggleDeletedEntities', deletedEntityCount.value) }}
+              <span v-show="deleted" class="icon-close"></span>
+            </button>
+          </template>
+          <odata-data-access :analyze-disabled="deleted"
+            :analyze-disabled-message="$t('analyzeDisabledDeletedData')"
+            @analyze="analyze.show()"/>
+          <entity-download-button
+            :aria-disabled="deleted"
+            v-tooltip.aria-describedby="deleted ? $t('downloadDisabled') : null"/>
```

Comment:
I think the message for `downloadDisabled` needs to be moved from `EntityList`. You can use `@transifexKey` to prevent the message from being retranslated after it's moved.